website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -16,6 +16,8 @@ description: |-
 
 -> **NOTE:** If Backup Compliance Policy is enabled for the project for which this backup schedule is defined, you cannot modify the backup schedule for an individual cluster below the minimum requirements set in the Backup Compliance Policy.  See [Backup Compliance Policy Prohibited Actions and Considerations](https://www.mongodb.com/docs/atlas/backup/cloud-backup/backup-compliance-policy/#configure-a-backup-compliance-policy).
 
+-> **NOTE:** When creating a backup schedule you **must either** use the `depends_on` clause to indicate the cluster to which it refers **or** specify the values of `project_id` and `cluster_name` as reference of the cluster resource (e.g. `cluster_name = mongodbatlas_cluster.my_cluster.name` - see the example below). Failure in doing so will result in an error when executing the plan.
+
```

Comment:
What about the feedback that the TSE shared with us today? (see https://mongodb.slack.com/archives/CLB2CJ1JB/p1694774725425429?thread_ts=1694106650.522989&cid=CLB2CJ1JB)

> But shouldn’t this resource see if there is a cluster named to what is supplied in the project and then execute, rather than starting execution and failing?

@Zuhairahmed Could you clarify which is the preferred option here?

Reply:
Actually, nvm my previous message. It makes sense to check for the cluster's status only if the `mongodbatlas_cloud_backup_schedule` fails when the cluster is in the 'IN_PROGRESS' state. If we are confident that the resource won't fail in that situation, displaying the error message is the best approach.

Reply:
>It makes sense to check for the cluster's status only if the mongodbatlas_cloud_backup_schedule fails when the cluster is in the 'IN_PROGRESS' state. If we are confident that the resource won't fail in that situation, displaying the error message is the best approach.

Sorry but I don't understand you here.

Reply:
The cluster has a status field which can have 4 values: `IN_PROGRESS`, `IDLE`, `DELETING` `UPDATING`. I was wondering what would happen if the cluster is not `IDLE` (the cluster is not "ready") and we use `mongodbatlas_cloud_backup_schedule`. I believe the resource won't fail as it will still find a cluster name

Reply:
I think if it doesn't fail it's the best case scenario. If it fails it will return an error (not a crash) that mentions the cluster was not found. For sure an alternative option would be to do what the TSE proposed but that also mean that we are adding one more call to the backend to do that. I would say this proposal should be fine and resolving customer's concerns.

Reply:
agree. Thanks for the discussion here